packages/radix-vue/src/Menubar/MenubarItem.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { inject } from "vue";
+import BaseMenuItem from "../shared/component/BaseMenuItem.vue";
+import { useArrowNavigation } from "@/shared/useArrowNavigation";
+import {
+  MENUBAR_INJECTION_KEY,
+  type MenubarProvideValue,
+} from "./MenubarRoot.vue";
+import {
+  MENUBAR_SUB_INJECTION_KEY,
+  type MenubarSubProvideValue,
+} from "./MenubarSub.vue";
+
+interface MenubarItemProps {
+  asChild?: boolean;
+  value?: string;
+  disabled?: boolean;
+  //onSelect?: void;
+  textValue?: string;
+}
+
+const rootInjectedValue = inject<MenubarProvideValue>(MENUBAR_INJECTION_KEY);
+
+const subInjectedValue = inject<MenubarSubProvideValue>(
+  MENUBAR_SUB_INJECTION_KEY
+);
+
+const props = defineProps<MenubarItemProps>();
+
+function handleClick() {
+  if (rootInjectedValue?.selectedElement.value) {
+    rootInjectedValue.hideTooltip();
+  }
+}
```

Comment:
this doesn't seems to close the menu dropdown. maybe `rootInjectedValue.setIsOpen(false)` ?